isort.cfg

```diff
@@ -22,4 +22,4 @@ default_section=THIRDPARTY
 known_first_party=readthedocs,readthedocsinc
 known_third_party=mock,builtins
 sections=FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
-add_imports=from __future__ import division, from __future__ import print_function, from __future__ import unicode_literals
+remove_imports=from __future__ import division, from __future__ import print_function, from __future__ import unicode_literals
```

Comment:
`remove_imports` :heart_eyes: 

(I didn't find it on its docs, though)

Reply:
https://github.com/timothycrosley/isort/blob/b5d7cd20696b7deed47cca3f2888c98fe34e5d50/isort/settings.py#L111